webpack/webpack.config.js

```diff
@@ -112,6 +112,7 @@ module.exports = {
           }
         ]
       },
+      { test: /\.flow$/, loader: "ignore-loader" },
```

Comment:
## 🔴 

why? 🤷‍♂️ 

i'd love to read about why this must be here, as i feel like we have to deeply understand everything in our webpack-config to be able to leverage opportunities of optimisation.

Reply:
The GraphiQL package loads some flow files which add external type definitions to the flow type checker used in that project. They are imported, but they mean nothing to us in our setup, but because the GraphiQL project is not compiled down to a single file or at least to plain javascript it brings these requires to `.flow` files. They can not be parsed by webpack, and ultimatively they don't need to, so I added an ignore loader that just lets them go to the void

Reply:
i don't like the burden they force on us.


> but because the GraphiQL project is not compiled down to a single file or at least to plain javascript

have you seen their integration-examples? isn't that a compiled js-file we could use as well? https://github.com/graphql/graphiql/blob/master/example/index.html#L40-L41

Reply:
Agree with @pierrebeitz I would love if we would not need to add something we don't need. This is unnecessary and error prone if someone cleans it up because they think it is a legacy thing which we don't need anymore.

Reply:
As discussed offline these examples don't improve the situation for us

Reply:
@Poltergeist I don't think not having them is possible without working on the upstream GraphiQL package

Reply:
TL;DR

let's try waiting for https://github.com/graphql/graphiql/issues/793 a bit. we should be able to get rid of this then.

-----------------------

it boils down to graphiQL using a version of `codemirror-graphql` that "accidentally" includes `graphql-language-service-interface` which in turn tried to include flow-files into our bundle. 

`graphql-language-service-interface` says "This is currently in technical preview. We welcome your feedback and suggestions.", so this is kind of expected. i feel JS-fatigue here :)